config/database.yml

```diff
@@ -27,30 +27,30 @@
 default: &default
   adapter:  postgresql
   encoding: unicode
-  username: <%= ENV.fetch('POSTGRES_USER',     nil) %>
-  password: <%= ENV.fetch('POSTGRES_PASSWORD', nil) %>
-  database: <%= ENV.fetch('POSTGRES_DB',       nil) %>
+  username: <%= ENV['POSTGRES_USER'].presence || nil %>
+  password: <%= ENV['POSTGRES_PASSWORD'].presence || nil %>
+  database: <%= ENV['POSTGRES_DB'].presence || nil %>
```

Comment:
To make sure i'm tracking: we're switching to `presence` because we want to ignore blank strings, whereas `fetch` would only ignore `nil`s, correct?

Reply:
💯That is correct. 

The idea  is that `presence` will return the contents of the ENV var when the ENV var contains what Ruby considers a truth-y value:

```ruby
# WUMPUS=1 
ENV['WUMPUS'].presence 
# => "1"

# WUMPUS=found_me 
ENV['WUMPUS'].presence  
# => "found_me"

# WUMPUS=true 
ENV['WUMPUS'].presence  
# => "true"

# WUMPUS=false
ENV['WUMPUS'].presence  
# => "false"
```

but will return `nil` when the ENV var is unset, or contains a false-y value:

```ruby
# WUMPUS=
ENV['WUMPUS'].presence  
# => nil

# WUMPUS=''
ENV['WUMPUS'].presence  
# => nil
```